### PR TITLE
Revert "bug fix: apply dropout to logits before softmax"

### PR DESCRIPTION
**Pull request by lfchener**
Revert "bug fix: apply dropout to logits before softmax"

### CODE REVIEW

**Review by iclementine — APPROVED** [review]
LGTM
